scripts/Two_VM_Deployment/.kube/config

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+clusters:
+- cluster:
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUN5RENDQWJDZ0F3SUJBZ0lCQURBTkJna3Foa2lHOXcwQkFRc0ZBREFWTVJNd0VRWURWUVFERXdwcmRXSmwKY201bGRHVnpNQjRYRFRFNU1ESXhNakUzTVRVek5Gb1hEVEk1TURJd09URTNNVFV6TkZvd0ZURVRNQkVHQTFVRQpBeE1LYTNWaVpYSnVaWFJsY3pDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTEFZCi9vWjNmQ1I5T2wzOURjN1Fzdlh3aExyM1p5RFJ6SVNXd3JqaWxvU0RJczFnUFR5SGxtTVdBM1Jmdk13TzA1ZVYKSzZ4dWhaaUd0b2drTGFOemN3ZnZ4M21iSUNzdWFZY3VBWXd0eklRdFNUd3pyMXpLOVliOVEzMWQwMmR6S25FTAp1anFTWVF6citPRWt2Zmlnc3VZaXgrY1cxNTVBRHV4UElCTXlMRndpV2REc05SYWRCWFRSRktDTHQ5dytyZm12Ck1icVV3NVhySnNqOGo1RHBCSTFYbEM5c3dhVXpGNktZY1NodUhyNkNLbmllbGJ6M3grdWVXRDUrZVFwZStKSnYKSE9mZ1JWcU15VVVXSHJPcHVyMnp4OTYyYnZtZEtCcUh2dkpiN2xLZER3UzJXbTc5TXpuTGVkWXo5Q2dxNU9pNApUUFFGQVhQYjk1bFVlaHhzVHowQ0F3RUFBYU1qTUNFd0RnWURWUjBQQVFIL0JBUURBZ0trTUE4R0ExVWRFd0VCCi93UUZNQU1CQWY4d0RRWUpLb1pJaHZjTkFRRUxCUUFEZ2dFQkFLRVdtSjZBaHpCandvdnE4MjVLVDRlcERSNHAKcDZmVWpnZmFtWlVCN2FEc0pyUm12YjlPZnRDbjU2N1hOdlE3TXN4ZU5pekNMaFNucS9RaDhETVlocHdscGorSwp4T2poVC9GdTMrOTBueVIzS2xTM0lESjFSZ3J1RndNZVRvSncyUXRNY3UzOVA1K2UxZ0QwcURpeEg4QlJaRVV6Clp4TGthVk4xTURCM2hnZzExK2drc09tSFhUSlJJR0pGRlVvSGkzQXRMRm9pK0JMZnM1RWpPMEhxUjRrVkNyQjgKOVdvbnpKU3NBS1dWRXdUNldaOVVMMnpwOU1UaHo4aktqWUIzZllueWk1NHlDL2pSZmNZNS9aNE5hOWFjd0JIZwpsbWlQUCtoZ3hTeHM2SWlkTkNjbWpjWGpJN05XTURXdTc2cmdIVjhQV29GZnFxcll1c202M3FYV3VwUT0KLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
+    server: https://172.31.16.96:6443
```

Comment:
How is this IP generated?

Reply:
It is the IP generated in the file /etc/kubernetes/admin.conf.
Indeed, we the file .kube/config is a copy of /etc/kubernetes/admin.conf, in order to enable the k8s cluster.

Reply:
But is that stable and reproducible? We have the option to use VBox, KVM and VMware Worksation/Fusion. Are they all enabling the same IP?

Reply:
I don't think that the IP would be the same.
I think it is better to remove the file .kube/config, because already the script copies its content from /etc/kubernetes/admin.conf. The latter depends on the VM on which k8s is installed I think.